### ISSUE TITLE
Xcode 11.4 compilation issue.

### ISSUE THREAD

**Issue by randomiser** [SwiftyTesseract/SwiftyTesseract#52]
Hi there,

As of Xcode 11.4 this library no longer compiles (despite the swift version being set at 4.2). 

The main issue is this:

```
  public func performOCR(on image: UIImage, completionHandler: @escaping (String?) -> ()) {
    let _ = semaphore.wait(timeout: .distantFuture)
    
    // pixImage is a var because it has to be passed as an inout paramter to pixDestroy to release the memory allocation
    var pixImage: Pix
    
    defer { <---- This block needs to be moved below the "do".
      // Release the Pix instance from memory
      pixDestroy(&pixImage)
      semaphore.signal()
    }
    
    do {
      pixImage = try createPix(from: image)
    } catch {
      completionHandler(nil)
      return
    }

   [I.e. pasted in here].
```

Afterwards it will compile and run fine.

Is that something we can have changed soon? Hope that's possible.

Thanks heaps in advance! 
Matt

**Comment by Steven0351** [comment]
What version of the library are you using? There was an issue with cocoa pods where this would occur and that was updated with version 2.2.2

**Comment by randomiser** [comment]
Thanks for the quick reply.

I'm using cocoapods 1.9.1 and SwiftyTesseract 2.2.2

**Comment by randomiser** [comment]
<img width="663" alt="Version 2 2 2" src="https://user-images.githubusercontent.com/2062188/77596297-84932200-6f60-11ea-87af-203f9398355d.png">

**Comment by MarcoFilosi** [comment]
Same here. What if we move `defer` block after `do-catch` block? It fixes it

**Comment by MarcoFilosi** [comment]
Another approach to make it compile, avoiding to move code, is to modify line 158 and declare `pixImage` as `Pix!`:

```
var pixImage: Pix!
```

What do you think?

**Comment by Steven0351** [comment]
This should be fixed in 2.2.3. Please reopen if the issue still persists after updating

**Comment by randomiser** [comment]
Thanks heaps Steven. All working now :)
